### ISSUE TITLE
Detects but doesn't block mining on techinferno

### ISSUE THREAD

**Issue by AjvarXX** [keraf/NoCoin#161]
https://www.techinferno.com/index.php?/forums/

This filthy site only wrote about mining in agreement after this filth was exposed. Also it mines even before you agreed on those terms.

**Comment by Firminator** [comment]
Confirmed. After connecting to coinhive CPU spikes from 5% to ~60%

**Comment by keraf** [comment]
I checked today and it looks like it's using the coinhive.com script which is already blocked by No Coin.

**Comment by AjvarXX** [comment]
NoCoin icon sais it blocked it but CPU usage jump is self-explanatory of what is going on.

**Comment by keraf** [comment]
I do also see my CPU jump pretty high while on this website but I haven't found any miners except coinhive which is already blocked. The dev console indicates that coinhive has been properly blocked. I ran a quick search through all the javascript but haven't found anything suspicious. Checked also all the iframes but no sign of mining there either. Might be a CPU hungry website or maybe I'm missing something?

**Comment by Firminator** [comment]
Anyone else has any insight on what is going on on this site?

**Comment by ryanbr** [comment]
I didn't notice any substantial hit to the cpu here

**Comment by keraf** [comment]
I'll close this issue as we haven't found any miners that wasn't blocked on that website. If you find a miner on some website that isn't blocked by No Coin, please open a new issue. Cheers!
